cylc/flow/scripts/install.py

```diff
@@ -154,14 +154,14 @@ def get_option_parser():
             dest="defines"
         )
         parser.add_option(
-            "--define-workflow", "--define-flow", '-S',
+            "--rose-template-variable", '-S',
             help=(
                 "As `--define`, but with an implicit `[SECTION]` for "
                 "workflow variables."
```

Comment:
Feels like we should change the description, and possibly the name of the `--define` option?

Reply:
I suppose `--rose-define` would be acceptable. Not worried.